### ISSUE TITLE
mrstft loss for each channel of 4 channel audio

### ISSUE THREAD

**Issue by ctemp48** [Stability-AI/stable-audio-tools#159]
Hello, I am adapting the "stable_audio_1_0_vae.json" training config file to train the DAC autoencoder on 4 channel audio. I have mostly figured it out, but am confused about one part, the loss. I want to find spectral loss by calculating the mrstft loss of each channel individually. Im not sure if this is possible as is or if I will have to modify the source code. Thank you!

**Comment by zqevans** [comment]
This would require modification of the training code. Right now it only supports channel-specific STFT losses for mono and stereo audio.
